examples/00-load/create-platonic-solids.py

```diff
@@ -0,0 +1,74 @@
+"""
+.. _platonic_example:
+
+Platonic Solids
+~~~~~~~~~~~~~~~
+
+PyVista wraps the ``vtk.vtkPlatonicSolid`` filter as
+:func:`pyvista.PlatonicSolid`.
+"""
+import numpy as np
+import pyvista as pv
+from pyvista import examples
+
+###############################################################################
+# We can either use the generic :func:`PlatonicSolid() <pyvista.PlatonicSolid>`
+# and specify the different kinds of solids to generate, or we can use the thin
+# wrappers
+#
+#     * :func:`pyvista.Tetrahedron`
+#     * :func:`pyvista.Octahedron`
+#     * :func:`pyvista.Dodecahedron`
+#     * :func:`pyvista.Icosahedron`
+#
+# (:func:`PlatonicSolid() <pyvista.PlatonicSolid>` can also return a cube, but
+# PyVista's existing :func:`pyvista.Cube` helper isn't based on the
+# ``vtkPlatonicSolid`` filter.)
```

Comment:
We could consider refactoring this.  It's up to you.  Ideally we would only have one way of generating a Cube.

Reply:
I'd be inclined to just replace `pv.Cube` with this one. Then again a `CubeSource` is as good a cube as any. I'd be worried that changing the implementation would change the order of points (although we already need `pv.Cube().clean()`...) which might subtly mess up downstream users doing weird things. I guess we could carefully add back the scalars, watching out to use the same order as what would come from `PlatonicSolidSource`.

Reply:
Implementation still isn't v1, so I think adding a `versionchanged` directive should be sufficient.

Reply:
OK, thanks. I now see that `pv.Cube()` has normals and texture coordinates defined. I'll try to figure out which change would be easier/more reasonable.

Reply:
Fixed in https://github.com/pyvista/pyvista/pull/1692/commits/51952c928a939013e756844ed63146665012492c.

I've decided to keep `pv.Cube()` and add `FaceIndex` to it. I also added a `clean` kwarg, like we did for the parametric objects. One wart with this is that TCoords are defined for points, and cleaning messes these up. But since the TCoords are undocumented, I figured we can afford to change the default to a cleaned version. I've added a note that if someone wants to use TCoords, they should pass `clean=False`.